scripts/device-farm/devicefarm.py

```diff
@@ -319,7 +319,6 @@ def schedule_run(self, app_package_arn, test_package_arn, test_spec_arn, outputF
 
         aws_command = ['aws', 'devicefarm', 'schedule-run',
                                             '--project-arn', self.project_arn,
-                                            '--configuration', 'customerArtifactPaths={iosPaths=["Documents"]}',
```

Comment:
Why is this removed? Can we re-add? This allows us to pull artifacts from the applications, though admittedly we don't have any yet.